Resources/skeleton/dto.php.twig

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * DTO {{ element.ucfirstName }}
+ */
+namespace {{ namespace }};
+
+use JMS\Serializer\Annotation;
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * {{ element.ucfirstName }}
+ * 
+ * @Annotation\ExclusionPolicy("none")
+ * @Annotation\XmlRoot("{{ element.name }}")
+ */
+class {{ element.ucfirstName }}
+{
+    {% for node in element.children %}
+/**
+     * @var {{ node.dataType }}
+     * @Annotation\Type("{{ node.dataType }}")
+     * @Annotation\SerializedName("{{ node.name }}")
+     * @Annotation\XmlElement(cdata=false)
```

Comment:
It would be a good addition to read the XSD for CDATA and add it conditionally here if configured for the node.